$MirOS: ports/mailnews/pine/patches/patch-imap_src_c-client_utf8_c,v 1.2 2005/05/04 23:37:39 tg Exp $
--- imap/src/c-client/utf8.c.orig	Fri Jan 18 00:08:55 2008
+++ imap/src/c-client/utf8.c	Wed May  7 20:53:25 2008
@@ -114,6 +114,8 @@ static const struct utf8_eucparam ksc_pa
 static const CHARSET utf8_csvalid[] = {
   {"US-ASCII",CT_ASCII,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
    NIL,NIL,NIL},
+  {"ISO_646.IRV:1991",CT_ASCII,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
+   NIL,NIL,NIL},
   {"UTF-8",CT_UTF8,CF_PRIMARY | CF_DISPLAY | CF_POSTING,
    NIL,SC_UNICODE,NIL},
   {"UTF-7",CT_UTF7,CF_PRIMARY | CF_POSTING | CF_UNSUPRT,
@@ -296,6 +298,8 @@ static const CHARSET utf8_csvalid[] = {
   {"IBM367",CT_ASCII,CF_PRIMARY | CF_DISPLAY,
      NIL,NIL,"US-ASCII"},
   {"IBM437",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
+     (void *) ibm_437tab,SC_LATIN_1,"ISO-8859-1"},
+  {"CP437",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
      (void *) ibm_437tab,SC_LATIN_1,"ISO-8859-1"},
   {"IBM737",CT_1BYTE,CF_PRIMARY | CF_DISPLAY,
      (void *) ibm_737tab,SC_GREEK,"ISO-8859-7"},
